change/@fluentui-cra-template-40467b48-9fa0-48c0-a783-7a3878481a16.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This actually should be `none` since it only affects tests for the template (not the template itself)